content/en/_index.md

```diff
@@ -293,14 +412,29 @@ sample) it is acceptable to either log or leave a comment explaining what the
 developer should do.
 
 {{< tabpane langEqualsHeader=true >}}
- {{< tab header="Java" >}}
+  {{< tab header="Java" >}}
     // Follow the Google Java style guide and catch the most specific type of Exception, instead of a more general one.
     try {
       // Do something
     } catch (IllegalArgumentException ok) {
       // IllegalArgumentException's are thrown when an invalid argument has been passed to a function. Ok to ignore.
     }
-{{< /tab >}}
+  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Unhandled exceptions will cause Node.js to crash. Print exception
+    // messages via console.error(), and handle any non-fatal exceptions.
```

Comment:
In Node.js, I think log or write are used more commonly, as `print` isn't really in a stdlib term.

```suggestion
    // Unhandled exceptions will cause Node.js to crash. Log exception
    // messages via console.error(), and handle any non-fatal exceptions.
```

Reply:
Done.